jscomp/others/js_dict.ml

```diff
@@ -27,6 +27,7 @@ type 'a t
 type key = string
 
 external get : 'a t -> key -> 'a option = "" [@@bs.get_index] [@@bs.return {undefined_to_opt}]
+external unsafe_get : 'a t -> key -> 'a = "" [@@bs.get_index] 
```

Comment:
Calling `Array.unsafe_get` in debug mode in OCaml does not cause an out of bounds exception? I find that quite horrifying if true. It's pretty much asking for heartbleed bugs.